views/default/tab_client_actions.pdt

```diff
@@ -1,4 +1,4 @@
-    <div id="change_password">
+    <div class="row" id="change_password">
```

Comment:
The addition of this class seems to really mess up the display for me.
![whmsonic-actions-tab](https://user-images.githubusercontent.com/18198499/100809634-659c3800-33eb-11eb-8769-50bba0af4376.png)